### PR TITLE
Save comment when user clicks "submit"

### PR DESCRIPTION
**Pull request by xtine**
For eregs/notice-and-comment#298, the comment is saved when the "Review and Submit" button is clicked even if the user hasn't clicked save.

### CODE REVIEW

File: regulations/static/regulations/js/source/views/comment/comment-view.js [hunk 1]
```diff
@@ -243,6 +249,14 @@ var CommentView = Backbone.View.extend({
...
     this.model.save();
     this.$status.hide().html('Your comment was saved.').fadeIn();
+  },
+
+  // when "Review and Submit" is clicked, save any unsaved content
+  // then open link on button
+  submitComments: function(e) {
+    this.save();
+
+    document.location.href = window.location.origin + $(e.target).attr('href');
```
Comment (cmc333333) [inline regulations/static/regulations/js/source/views/comment/comment-view.js]: This feels wonky -- we're preventing the event from completing, sending an event, and then "manually" completing the same action. I know a lot of this indirection stems from Backbone's architecture, but is there a more direct way to accomplish what we need?

**Comment by cmc333333** [comment]
Hm. We save empty comments, which we probably shouldn't, but is more prominent with this change. Fixing that (not saving empty comments) opens up a bigger can of worms though (we'd need error messages, figure out if blanking out a comment means deleting the original comment, etc.). I'd recommend we punt on that given our limited runway. Is there a simpler solution? Maybe we just redesign that link so it doesn't look like a button?

**Comment by cmc333333** [comment]
Per https://github.com/eregs/notice-and-comment/issues/298#issuecomment-224630338 , we'll punt for now. Is there anything in this PR which we should merge anyway, @xtine?

**Comment by xtine** [comment]
I'm fine with punting this, there isn't anything else that needs to be merged in.

**Comment by cmc333333** [comment]
:+1: closing

**Pull request closed without merging**